tests/greenboot_check_fail_wanted.bats

```diff
@@ -10,5 +10,5 @@ function setup() {
 }
 
 function teardown() {
-    rm $GREENBOOT_CHECK_PATH/wanted.d/10*.sh    
+    rm $GREENBOOT_CHECK_PATH/wanted.d/10*.sh
 }
```

Comment:
please make sure to add new lines to all files where they are currently missing

Reply:
?